engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/services/ExecutionStateServiceImpl.java

```diff
@@ -160,6 +160,17 @@ public void deleteCanceledExecutionStates() {
         }
     }
 
+    @Override
+    public Execution getExecutionObjectForNullBranch(Long executionId) {
+        validateExecutionId(executionId);
+        ExecutionState executionState = readByExecutionId(executionId).get(0);
+        if (executionState.getExecutionObject() != null) {
+            return executionSerializationUtil.objFromBytes(executionState.getExecutionObject());
+        } else {
+            return null;
+        }
+    }
```

Comment:
The method name doesn't really reflect what's happening.. Why do we need a method for null branch ? the readByExecutionId is not enough ?